docs/lua/constants.md

```
@@ -36,6 +36,7 @@
 - [gfx_pc.h](#gfx_pch)
     - [enum ShaderFlag](#enum-ShaderFlag)
 - [graph_node.h](#graph_nodeh)
+    - [enum GraphNodeType](#enum-GraphNodeType)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid
Expected: `#enum-graphnodetype`; Actual: `#enum-GraphNodeType`

(MD051, link-fragments)